src/app/Shared/Services/api.utils.ts

```diff
@@ -424,4 +424,58 @@ export const messageKeys = new Map([
       body: (evt) => `Credentials deleted for: ${evt.message.matchExpression}`,
     } as NotificationMessageMapper,
   ],
+  [
+    NotificationCategory.ReportSuccess,
+    {
+      variant: AlertVariant.info,
+      title: 'Report Success',
+      body: (evt) => `Report generated successfully for job: ${evt.message.jobId}`,
+      hidden: true,
+    } as NotificationMessageMapper,
+  ],
+  [
+    NotificationCategory.ReportFail,
+    {
+      variant: AlertVariant.warning,
+      title: 'Report Generation Failed',
+      body: (evt) => `Report generation failed for job: ${evt.message.jobId}`,
+      hidden: true,
+    } as NotificationMessageMapper,
+  ],
+  [
+    NotificationCategory.GrafanaUploadSuccess,
+    {
+      variant: AlertVariant.success,
+      title: 'Grafana Upload Success',
+      body: (evt) => `Recording successfully uploaded to Grafana for job: ${evt.message.jobId}`,
+      hidden: false,
+    } as NotificationMessageMapper,
+  ],
+  [
+    NotificationCategory.GrafanaUploadFail,
+    {
+      variant: AlertVariant.warning,
+      title: 'Grafana Upload Failed',
+      body: (evt) => `Grafana upload failed for job: ${evt.message.jobId}`,
+      hidden: false,
+    } as NotificationMessageMapper,
+  ],
+  [
+    NotificationCategory.ArchiveRecordingSuccess,
+    {
+      variant: AlertVariant.warning,
```

Comment:
![image](https://github.com/user-attachments/assets/9cbe5bf1-4c0e-4c8a-8726-1bcbc7c28bda)

This seems like it should be a non-hidden success message, not a hidden warning